src/server/auth/auth.js

```diff
@@ -1,9 +1,17 @@
+// @flow
+
 import 'dotenv/config';
 import passport from 'passport';
 
 import { Strategy as GithubStrategy } from 'passport-github';
 
+import type { Strategy } from 'passport';
+
 class Authentication {
+  passport: passport;
+
+  githubConfig: any;
```

Comment:
Lets group fields together? The space here does not aid readability IMO. Please address throughout

Reply:
Yeah I thought this looked weird too, ESLint gets pissy if I don't have it like this. I'll add a new ESlint rule to deal with it.